### PR TITLE
Consolidate BMS CAN Messages

### PR DESCRIPTION
**Pull request by will-chaba**
### Changelist 

Consolidate BMS CAN Messages, including the deletion of un-used debugging messages to reduce bus load and ease burden on data logging

- Combined signals where possible
- Reduced cycle time where possible

### Testing Done

- Existing unit tests updated to match new signals

### CODE REVIEW

File: can_bus/quadruna/BMS/BMS_tx.json [hunk 1]
```diff
@@ -259,7 +252,7 @@
         "description": "BMS Tractive System Data",
         "signals": {
             "TractiveSystemVoltage": {
-                "resolution": 0.0001,
+                "resolution": 0.01,
```
Comment (will-chaba) [inline can_bus/quadruna/BMS/BMS_tx.json:255]: our voltage sense is not accurate enough to warrant this resolution

File: can_bus/quadruna/BMS/BMS_tx.json [hunk 2]
```diff
@@ -271,82 +264,19 @@
...
             "AvailablePower": {
-                "resolution": 0.1,
+                "resolution": 1,
                 "min": 0,
                 "max": 100000,
                 "unit": "W"
             }
         }
     },
-    "AuxThermistors": {
```
Comment (will-chaba) [inline can_bus/quadruna/BMS/BMS_tx.json:307]: these are not currently installed, can be added back later for future testing

File: firmware/quadruna/BMS/src/app/app_thermistors.c [hunk 3]
```diff
@@ -28,10 +28,11 @@ void app_thermistors_updateAuxThermistorTemps()
...
-    app_canTx_BMS_ThermTemp3_set(therm_temps[3]);
-    app_canTx_BMS_ThermTemp4_set(therm_temps[4]);
-    app_canTx_BMS_ThermTemp5_set(therm_temps[5]);
+    // Re-enable if aux thermistors installed 
+    // app_canTx_BMS_ThermTemp0_set(therm_temps[0]);
+    // app_canTx_BMS_ThermTemp1_set(therm_temps[1]);
+    // app_canTx_BMS_ThermTemp2_set(therm_temps[2]);
+    // app_canTx_BMS_ThermTemp3_set(therm_temps[3]);
+    // app_canTx_BMS_ThermTemp4_set(therm_temps[4]);
+    // app_canTx_BMS_ThermTemp5_set(therm_temps[5]);
```
Comment (will-chaba) [inline firmware/quadruna/BMS/src/app/app_thermistors.c:37]: Aux Thermistors not currently installed, no sense loading the CAN bus with their messages

Comment (gtaharaedmonds) [reply firmware/quadruna/BMS/src/app/app_thermistors.c:37]: You should delete them from the JSON instead, commenting out the setters doesn't disable them from the bus. They'll just be transmitted with their start value (probably all zeros).

File: scripts/canup
Comment (gtaharaedmonds) [inline scripts/canup:1]: Was this on purpose?

**Review by gtaharaedmonds — CHANGES_REQUESTED** [review]
Yeep this is definitely an improvement

**Review by Lucien950 — APPROVED** [review]
i am happy when gus is happy

File: scripts/canup
Comment (will-chaba) [reply scripts/canup:1]: No definitely not, will revert

File: firmware/quadruna/BMS/src/app/app_thermistors.c (hunk 3 above)
Comment (will-chaba) [reply firmware/quadruna/BMS/src/app/app_thermistors.c]: Ah okay gotcha. Will do.

**Review by gtaharaedmonds — APPROVED** [review]
yeep

**Pull request merged**